dyn_load.c

```diff
@@ -714,8 +714,9 @@ GC_FirstDLOpenedLinkMap(void)
 
         for( dp = _DYNAMIC; (tag = dp->d_tag) != 0; dp++ ) {
             if( tag == DT_DEBUG ) {
-                struct link_map *lm
-                        = ((struct r_debug *)(dp->d_un.d_ptr))->r_map;
+                struct r_debug *rd = (struct r_debug *)(dp->d_un.d_ptr);
+                if ( !dp->d_un.d_ptr ) continue;
```

Comment:
Does it worth to have "continue" in this case? (Do you see other elements with DT_DEBUG tag that have non-null d_ptr?)